CRM/MembershipExtras/Upgrader.php

```diff
@@ -179,4 +179,15 @@ public function upgrade_0001() {
     return TRUE;
   }
 
+  /**
+   * Renames membershipextras_contribution_recur_line_item table to membershipextras_subscription_line;.
+   *
+   * @return bool
+   */
+  public function upgrade_0002() {
```

Comment:
There's no need for this upgrade, per Guanhuan's instructions. The idea is to have only one upgrader for the whole of phase 3 (that would be upgrade_0001), and since we've done the changes on the auto_install.sql file, it is enough. This will require you and me to uninstall the extension and re-install it to keep on working, but that's the idea.

Reply:
@guanhuan mentioned that people are already using the extension